### PR TITLE
Add functionality to get fresh state on new connection

### PR DESCRIPTION
**Pull request by canlasla**
PR implements how the server handles communication between a new client and an already connected client. The idea is to get the state from the already connected client and send it to the new client to process. Here are the steps:

When the client first starts up, call the `ReceiveFreshState` hub on the server. This will select a client to send a state to the new client.

The server will invoke `SendMessageToNewConnection` on the selected client. This will make the selected client grab its current state and call the `SendServerCurrentState` hub on the server. 

The server will receive this state and invoke `ReceiveEncodedMessage` on the new client for it to process.

### CODE REVIEW

**Comment by canlasla** [comment]
[link to pr on connectionserver repo](https://github.com/fontan10/ConnectionServer/pull/1 ) @fontan10 @ayshon

File: MergeSharpWebAPI/ServerConnection/Globals.cs [hunk 1]
```diff
@@ -27,8 +27,8 @@ public static class Globals
 
     //public static IDMapping = new Dictionary<Guid, int>() { { Guid.NewGuid(), 1}, { Guid.NewGuid(), 2 } };
 
-    private const string PropagationMessageServer = "https://serverwebapi20230125175127.azurewebsites.net/hubs/propagationmessage";
-    // private const string PropagationMessageServer = "https://localhost:7106/hubs/propagationmessage";
+    // private const string PropagationMessageServer = "https://serverwebapi20230125175127.azurewebsites.net/hubs/propagationmessage";
+    private const string PropagationMessageServer = "https://localhost:7106/hubs/propagationmessage";
```
Comment (fontan10) [inline MergeSharpWebAPI/ServerConnection/Globals.cs]: this change should not be merged to master

**Review by ayshon — APPROVED** [review]
Looks good sir

**Review by fontan10 — CHANGES_REQUESTED** [review]

File: MergeSharpWebAPI/Program.cs [hunk 2]
```diff
@@ -87,48 +87,59 @@ private static void ConfigureServerConnectionOnReceiveEncodeMessage()
         {
             //Console.WriteLine("Message received: ", byteMsg);
 
-            //Declare TPTPMsg
-            MergeSharp.TPTPGraphMsg tptpgraphMsg = new MergeSharp.TPTPGraphMsg();
-            tptpgraphMsg.Decode(byteMsg);
+            // //Declare TPTPMsg
```
Comment (fontan10) [inline MergeSharpWebAPI/Program.cs]: if youre able to decide on a block to comment out, you can factor it into a function. then just comment out the funciton call

File: MergeSharpWebAPI/Program.cs [hunk 3]
```diff
@@ -81,54 +81,76 @@ private static void ConfigureServerConnectionClosed()
...
+            // ProcessAndDisplayTptpGraphMsg(byteMsg);
+            ProcessAndDisplayLwwSetMsg(byteMsg);
         });
     }
 
+    private static void ConfigureServerConnectionOnSendMessageToNewConnection()
+    {
+        _ = serverConnection.On<string>("SendMessageToNewConnection", async newConnectionID =>
+        {
+            Console.WriteLine(JsonConvert.SerializeObject(myLWWSetService.Get(1)));
```
Comment (fontan10) [inline MergeSharpWebAPI/Program.cs:148]: can you add a TODO: here to change it to the GraphService later

**Review by fontan10 — APPROVED** [review]
good to merge

**Pull request merged**